packages/fiori/src/ShellBarTemplate.tsx

```diff
@@ -0,0 +1,275 @@
+import Icon from "@ui5/webcomponents/dist/Icon.js";
+import Button from "@ui5/webcomponents/dist/Button.js";
+import type ShellBar from "./ShellBar.js";
+import ShellBarPopoverTemplate from "./ShellBarPopoverTemplate.js";
```

Comment:
we also import the icons in the template. each icon provide default export that has type string with the icon name and we save it in named import which is used later on in the template to ensure that all used icons are imported.